docs/flists/router.md

```diff
@@ -0,0 +1,81 @@
+# Block routing
```

Comment:
this file doesn't seems to be link from anywhere in the doc. so unless you manually look in the repo you'll never find this file